.travis.yml

```diff
@@ -2,6 +2,11 @@ language: php
 
 sudo: false
 
+cache:
+    directories:
+        - vendor
```

Comment:
this can lead to weird things in composer. If the goal is to cache the PHPUnit download, put it outside `vendor` IMO